### ISSUE TITLE
No Amazon Bedrock models can be used as embedding providers for similarity assertion

### ISSUE THREAD

**Issue by markstownsend** [promptfoo/promptfoo#795]
Hello,
No Amazon Bedrock models can be used as embedding providers for similarity assertion.  I have tried all of them with these results:

``` sh
Provider bedrock:cohere.embed-english-v3 is not a valid embedding provider for 'similarity check', falling back to default
```
``` sh
Provider bedrock:amazon.titan-embed-text-v2:0 is not a valid embedding provider for 'similarity check', falling back to default
```
``` sh
Provider bedrock:amazon.titan-embed-text-v1 is not a valid embedding provider for 'similarity check', falling back to default
```
If it is a new feature rather than my mistake and, you think its doable for a newbie I'm happy to try and create a PR for it.  It seems I would need to implement ```BedrockEmbeddingProvider``` class in [bedrock.ts](https://github.com/promptfoo/promptfoo/blob/bfaa9e9882fca26edaa4c9dd31de319b34e40855/src/providers/bedrock.ts).
Cheers,
Mark

**Comment by typpo** [comment]
#797 adds support for Bedrock embedding models!

**Comment by typpo** [comment]
Released in 0.59.0!
